components/common/ProfileInfoCard.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+  Flex,
+  Circle,
```

Comment:
Currently this page is not accessible from anywhere.
Pls add a link in navbar or any other place where user will click to reach Super-Shardians page..

Reply:
I will connect with product team for the same as it is not in design, for now adding it to navbar.